apps/web/pages/team/[slug].tsx

```diff
@@ -51,6 +54,7 @@ function TeamPage({ team }: TeamPageProps) {
             "hover:border-brand group relative rounded-sm border border-neutral-200   dark:border-neutral-700 dark:bg-neutral-800 dark:hover:border-neutral-600",
             isEmbed ? "" : "bg-white hover:bg-gray-50"
           )}>
+          {(type.description = md.render(`${type.description}`))}
```

Comment:
Making an assignment inside a render function is not considered a optimal practice